stix_shifter_modules/aws_athena/stix_translation/json/stix_2_1/to_stix_map.json

```diff
@@ -0,0 +1,3100 @@
+            },
+            "credential_uid": {
+              "key": "user-account.extensions.x-accessor-ext.credential_uid",
+              "object": "parent-modifier-user-account"
+            },
+            "domain": {
+              "key": "user-account.extensions.x-accessor-ext.domain",
+              "object": "parent-modifier-user-account"
+            },
+            "email_addr": {
+              "key": "email-addr.value",
+              "object": "parent-modifier-email-addr"
+            },
+            "groups": {
+              "desc": {
+                "key": "user-account.extensions.x-accessor-ext.group_desc",
+                "object": "parent-modifier-user-account"
+              },
+              "name": {
+                "key": "user-account.extensions.x-accessor-ext.group_name",
+                "object": "parent-modifier-user-account"
+              },
+              "privileges": {
+                "key": "user-account.extensions.x-accessor-ext.group_privileges",
+                "object": "parent-modifier-user-account"
+              },
+              "type": {
+                "key": "user-account.extensions.x-accessor-ext.group_type",
+                "object": "parent-modifier-user-account"
+              },
+              "uid": {
+                "key": "user-account.extensions.x-accessor-ext.group_uid",
+                "object": "parent-modifier-user-account"
+              }
+            },
+            "name": {
+              "key": "user-account.display_name",
+              "object": "parent-modifier-user-account"
+            },
+            "org_uid": {
+              "key": "user-account.extensions.x-accessor-ext.org_uid",
+              "object": "parent-modifier-user-account"
+            },
+            "session_uid": {
+              "key": "user-account.extensions.x-accessor-ext.session_uid",
+              "object": "parent-modifier-user-account"
+            },
+            "session_uuid": {
+              "key": "user-account.extensions.x-accessor-ext.session_uuid",
+              "object": "parent-modifier-user-account"
+            },
+            "type": {
+              "key": "user-account.extensions.x-accessor-ext.type",
+              "object": "parent-modifier-user-account"
+            },
+            "type_id": {
+              "key": "user-account.extensions.x-accessor-ext.type_id",
+              "object": "parent-modifier-user-account"
+            },
+            "uid": {
+              "key": "user-account.user_id",
+              "object": "parent-modifier-user-account"
+            },
+            "uuid": {
+              "key": "user-account.extensions.x-accessor-ext.uuid",
+              "object": "parent-modifier-user-account"
+            }
+          },
+          "name": [
+            {
+              "key": "file.name",
+              "object": "parent-file"
+            },
+            {
+              "key": "process.binary_ref",
```

Comment:
This is `process.image_ref` in STIX 2.1.

Reply:
updated